corehq/apps/es/client.py

```diff
@@ -812,6 +812,11 @@ def is_index(self):
         """``True`` if this is an index action, otherwise ``False``."""
         return self.op_type is self.OpType.index
 
+    def __eq__(self, other):
+        if not isinstance(other, BulkActionItem):
+            return False
```

Comment:
nit: I think this could/should return [`NotImplemented`](https://docs.python.org/3/library/constants.html#NotImplemented).

The docs somewhat confusingly say "If all attempts return `NotImplemented`, the interpreter will raise an appropriate exception." It will return `False` if both `__eq__` methods return that:
```py
>>> class A:
...     def __eq__(self, other):
...         return NotImplemented
... 
>>> class B:
...     def __eq__(self, other):
...         return NotImplemented
... 
>>> A() == B()
False
```

Reply:
Thanks, I've never dug into doing this elegantly, but have always been curious about the various "correct" things to do here.